src/tests/unit/stream_element_tests.js

```diff
@@ -177,7 +177,7 @@ test("test action=refresh", async () => {
   const element = createStreamElement("refresh")
   subject.append(element)
 
-  await nextBeat()
+  await sleep(250)
```

Comment:
Could we use `nextPageRefresh` instead of the remaining `sleep(250)`?

Reply:
That helper uses Playwright and we don't have access to it here, that's why I haven't changed it. But you're right that it doesn't read well.